moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -105,6 +105,70 @@ void MotionPlanningFrame::sceneScaleChanged(int value)
           s->scale((double)value / 100.0);
           ps->getWorldNonConst()->addToObject(scaled_object_->id_, shapes::ShapeConstPtr(s),
                                               scaled_object_->shape_poses_[i]);
+
+          // get shape bound size
+          double half_bound = -std::numeric_limits<double>::max();
+          switch(s->type){
+            case shapes::ShapeType::SPHERE:{
+              shapes::Sphere* sphere = static_cast<shapes::Sphere*>(s);
+              half_bound = sphere->radius;
+              break;
+            }
+            case shapes::ShapeType::CYLINDER:{
+              shapes::Cylinder* cylinder = static_cast<shapes::Cylinder*>(s);
+              half_bound = (cylinder->radius > cylinder->length/2.0) ? cylinder->radius : cylinder->length/2.0;
+              break;
+            }
+            case shapes::ShapeType::CONE:{
+              shapes::Cone* cone = static_cast<shapes::Cone*>(s);
+              half_bound = (cone->radius > cone->length/2.0) ? cone->radius : cone->length/2.0;
+              break;
+            }
+            case shapes::ShapeType::BOX:{
+              shapes::Box* box = static_cast<shapes::Box*>(s);
+              for(std::size_t i = 0; i < 3; i++)
+                half_bound = (half_bound < box->size[i]/2.0) ? box->size[i]/2.0 : half_bound;
```

Comment:
replace for loop with `half_bound = std::max_element(box->size.begin(), box->size.end())`?

Reply:
Same, I will replace here into computeShapeBoundingSphere() function.